src/test/java/picocli/ExecuteTest.java

```diff
@@ -480,15 +480,17 @@ public int getExitCode(Throwable t) {
         int exitCode = cmd.execute();
         assertEquals(1234, exitCode);
 
-        assertThat(this.systemErrRule.getLog(), startsWith("java.io.IOException: error"));
+        assertThat(this.systemErrRule.getLog(), startsWith(Help.Ansi.ON.new Text(format(
+                "@|fg(red),bold java.io.IOException: error|@"
+        )).toString()));
```

Comment:
Why does the stack trace contain ANSI escape codes, when the ExecuteTest switches off ANSI colors explicitly?

The below test rule defined in ExecuteTest ensures that system property `picocli.ansi` is set to `false` before each test. That should result in no ANSI escape codes being emitted. But you have changed the test expectations to expect ANSI escaped codes, and these tests pass... This is strange. Why is this?


```java
public class ExecuteTest {
    @Rule
    public final ProvideSystemProperty ansiOFF = new ProvideSystemProperty("picocli.ansi", "false");
```

Reply:
Sorry, don't notice this... I'll revert my changes to this file.

Reply:
It seems my original implementation don't check if ANSI is enabled or disabled. Current version uses Ansi.Text.toString, which will check it now.

Reply:
Changes to `ExecuteTest` reverted.